requirements/requirements-test.txt

```diff
@@ -9,3 +9,4 @@ flake8
 tox
 tox-pyenv
 wheel
+pandas
```

Comment:
@bryan-bisonai Pandas are already imported through commont.txt at https://github.com/Bisonai/datamaxi-python/blob/31a40e5198d30326e74e12a7b7da89f85631463a/requirements/common.txt#L2

Reply:
@martinkersner If I don't have pandas installed in local and follow the steps described in the readme.md, I get a complaint that pandas is not installed. 

pip3 install -r requirements/requirements-test.txt

python3 -m pytest tests/